index.js

```diff
@@ -52,15 +53,15 @@ function createSkynsEntry(entryUrl) {
     const rawSkylink = parseSkylink(skylink);
     const skylinkDecoded = decodeBase64(rawSkylink);
     const skylinkEncodedBase32 = encodeBase32(skylinkDecoded);
-    const skylinkUrl = `https://${skylinkEncodedBase32}.siasky.net`;
+    const skylinkUrl = `${portalUrl.protocol}//${skylinkEncodedBase32}.${portalUrl.host}`;
```

Comment:
we should use getSkylinkUrl method to get this link instead of constructing it ourselves

https://github.com/SkynetLabs/skynet-js/blob/master/src/download.ts#L180-L192

You wouldn't need to construct `new URL` then too